tests/fastapi/run_fast_test.py

```diff
@@ -0,0 +1,45 @@
+import uuid
```

Comment:
If you're gonna put the example at `fast_api`, then this dir should be `tests/fast_api`